### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by JohnMcIntyreJR**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,11 @@
 Type: Package
 Package: jrProgramming
 Title: Jumping Rivers: Introduction to Programming
+<<<<<<< HEAD
```
Comment (csgillespie) [inline DESCRIPTION:4]: This merge isn't correct

Comment (JohnMcIntyreJR) [reply DESCRIPTION:4]: Yep, forgot to alter DESCRIPTION @csgillespie

File: vignettes/practical1_contents.Rmd [hunk 2]
```diff
@@ -1,37 +1,29 @@
...
-    message(paste("The country", i, "consists of", nchar(i), "characters."))
-}
-``` 
+Re-order the lines of code below to create a for loop!
 
-The `for` loop above gives us information on the number of letters in a selection of ten European countries.
+    1. message(day)
 
-1. Try and run the `for` loop yourself.
+    2. days = c("Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday",                        "Sunday")
```
Comment (csgillespie) [inline vignettes/practical1_contents.Rmd:7]: Why the big gap?

File: vignettes/practical1_contents.Rmd [hunk 3]
```diff
@@ -1,37 +1,29 @@
...
-                       "Spain", "Italy", "Boznia and Herzegovina",
-                       "Croatia", "Montenegro", "Sweden")
-for (i in european_countries) {
-    message(paste("The country", i, "consists of", nchar(i), "characters."))
-}
-``` 
+Re-order the lines of code below to create a for loop!
 
-The `for` loop above gives us information on the number of letters in a selection of ten European countries.
+    1. message(day)
```
Comment (csgillespie) [inline vignettes/practical1_contents.Rmd:5]: Should be surrounded by backticks

Comment (JohnMcIntyreJR) [reply vignettes/practical1_contents.Rmd:5]: Okay

File: vignettes/practical1_contents.Rmd (hunk 2 above)
Comment (JohnMcIntyreJR) [reply vignettes/practical1_contents.Rmd:7]: Annoying markdown formatting, but i'll fix that

**Comment by jamieRowen** [comment]
Build checks currently failing due to missing news.md, happy to review once build passes @csgillespie @JohnMcIntyreJR

**Comment by JohnMcIntyreJR** [comment]
> Build checks currently failing due to missing news.md, happy to review once build passes @csgillespie @JohnMcIntyreJR

I can add one in if we have a template news.md file somewhere?  Seem to be a few changes to this package since i've been away, e.g. allowing 3 notes instead of just the vignette builder note

**Comment by jamieRowen** [comment]
Check out one of the other packages for the format. jrNotes has one

**Comment by csgillespie** [comment]
You don't need to do the complete NEWS history. Just your change
Dr Colin Gillespie
http://www.mas.ncl.ac.uk/~ncsg3/

**Comment by JohnMcIntyreJR** [comment]
Okay

**Comment by JohnMcIntyreJR** [comment]
> Does this not work:
> 
> \noindent Try changing `day` to `i` wherever...

Yep it does, can change that back as well. I got used to using LaTeX formatting after \noindent when you converted some of the formatting to LaTeX.

**Pull request merged**